### PR TITLE
(GH-101) Correcting issue with detecting chocolatey tag

### PR DESCRIPTION
**Pull request by gep13**
Relates to #101

### CODE REVIEW

**Comment by gep13** [comment]
@ferventcoder I "think" I have captured everything for this one.

I originally set out to use a Regex for validation, but then I opted to split the tags up instead, and checked explicitly for "chocolatey".

File: src/chocolatey.package.validator/infrastructure.app/rules/TagsDoNotContainChocolateyRequirement.cs [hunk 1]
```diff
@@ -28,9 +29,11 @@ public class TagsDoNotContainChocolateyRequirement : BasePackageRule
         public override PackageValidationOutput is_valid(IPackage package)
         {
             if (string.IsNullOrWhiteSpace(package.Tags)) return true;
-            if (package.Id.to_lower().Contains("chocolatey")) return true;
+            if (package.Id.to_lower().Equals("chocolatey")) return true;
+
+            var tags = package.Tags.Split(' ');
 
-            return !package.Tags.to_lower().Contains("chocolatey");
+            return tags.All(tag => !tag.Equals("chocolatey"));
```
Comment (ferventcoder) [inline src/chocolatey.package.validator/infrastructure.app/rules/TagsDoNotContainChocolateyRequirement.cs]: Still needs a case insensitive comparison.

Comment (gep13) [inline src/chocolatey.package.validator/infrastructure.app/rules/TagsDoNotContainChocolateyRequirement.cs]: Doh! Good catch. I think I was actually falling asleep on the train while doing this PR :-D

**Comment by gep13** [comment]
@ferventcoder fixed up.

**Pull request merged**